modules/axhal/src/platform/dummy/mod.rs

```diff
@@ -11,6 +11,17 @@ pub mod console {
     pub fn getchar() -> Option<u8> {
         unimplemented!()
     }
+
+    /// Writes bytes to the console from input u8 slice.
```

Comment:
Do we still need `putchar` and `getchar`?

Reply:
> Do we still need `putchar` and `getchar`?

This is the interface that was left over from before. I'll keep it if it doesn't affect me.